ethereum/blockwatch/block_watcher.go

```diff
@@ -129,6 +129,14 @@ func (w *Watcher) StopPolling() {
 	w.ticker = nil
 }
 
+// Stop stops the BlockWatcher
+func (w *Watcher) Stop() {
```

Comment:
Should `StopPolling` be unexported? It looks like things will break if you call it directly instead of via `Stop`.